src/ResourceManager/DataLakeStore/Commands.DataLakeStore/Commands/ExportAzureRmDataLakeStoreItem.cs

```diff
@@ -95,20 +116,59 @@ public override void ExecuteCmdlet()
                         throw new CloudException(string.Format(Resources.InvalidExportPathType, Path.TransformedPath));
                     }
 
-                    if (type == FileType.FILE)
+                    DataLakeStoreTraceLogger logger = null;
```

Comment:
Agreed. This trace logging is a "first step" to enable some of our internal and external partners to get initial heuristics about upload and download operations (mostly to help us with performance tuning). As this matures we will definitely need to update this and make it more robust. Ideally, we are able to integrate this into core AzurePowerShell functionality so that everyone can take advantage of it.